batch/sql/insert_globals.py

```diff
@@ -10,7 +10,9 @@
 
     worker_type = 'standard'
     if scope == 'deploy':
-        worker_cores = 8
+        # warning: if you change worker_cores or worker_disk_size_gb,
+        # make sure to change batch/utils.py::cost_from_msec_mcpu
+        worker_cores = 16
```

Comment:
@cseed FYI: I'm changing the worker cores to 16 here.

Reply:
I propose we take this one step fruther.  The cores, worker type and disk size are all stored in the batch global configuration.  We already pull out the cores and worker type and store them in the app  configuration.  From front_end.py:

```
    row = await db.select_and_fetchone(
        'SELECT worker_type, worker_cores, instance_id, internal_token FROM globals;')

    app['worker_type'] = row['worker_type']
    app['worker_cores'] = row['worker_cores']
```

Can we pull out the disk size, too, and use it in the cost calculation?  I think for now you can assert the type is standard and so 0.01/core.  Then you can calculate the disk cost per core from the disk size and the number of cores.  Then we can change the config and this will remain correct (and eventually take these things from the instance instead of being global).  Thanks!